ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml

```diff
@@ -181,25 +187,44 @@ SettingsPageLayout {
         id: collectibleView
 
         CommunityCollectibleView {
+            id: collectibleViewComponent
+
             viewWidth: root.viewWidth
             preview: d.preview
```

Comment:
What about passing it via `stackManager.push` as other parameters instead of via `d` and states?